src/components/Logo/__tests__/Logo.test.js

```diff
@@ -22,30 +24,30 @@ describe('components/Logo', () => {
   });
 
   it('should render correctly', () => {
-    const { container } = render(withAppContext(<Logo />));
+    const { container, getByTestId } = render(withAppContext(<Logo tall />));
```

Comment:
`tall` Defaults to 'true'. No need to provide it.

Reply:
IMHO this is more explicit, but will remove.

Reply:
Did you?